packages/react/src/components/Storage/FileUploader/Previewer/index.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { ComponentClassName, translate } from '@aws-amplify/ui';
+import { PreviewerProps } from '../types';
+import {
+  Alert,
+  Button,
+  ComponentClassNames,
+  Loader,
+  Text,
+  View,
+} from '../../../../primitives';
+import { UploadDropZone } from '../UploadDropZone';
+import { UploadButton } from '../UploadButton';
+
+export function Previewer({
+  acceptedFileTypes,
+  children,
+  fileStatuses,
+  inDropZone,
+  isLoading,
+  isSuccess,
+  maxFilesError,
+  multiple,
+  onClear,
+  onDragEnter,
+  onDragLeave,
+  onDragOver,
+  onDragStart,
+  onDrop,
+  onFileChange,
+  onFileClick,
+  percentage,
+  hiddenInput,
+  onUploadButtonClick,
+}: PreviewerProps): JSX.Element {
+  const headingMaxFiles = translate('Over Max files!');
+  const uploadedFilesLength = () =>
+    fileStatuses.filter((file) => file?.fileState === 'success').length;
+
+  const remainingFilesLength = fileStatuses.length - uploadedFilesLength();
+  return (
+    <View className={ComponentClassNames.FileUploaderPreviewer}>
+      <View className={ComponentClassNames.FileUploaderPreviewerBody}>
+        <UploadDropZone
+          inDropZone={inDropZone}
+          onDragEnter={onDragEnter}
+          onDragLeave={onDragLeave}
+          onDragOver={onDragOver}
+          onDragStart={onDragStart}
+          onDrop={onDrop}
+        >
+          <Text className={ComponentClassNames.FileUploaderDropZoneText}>
+            {translate('Drop files here or')}
+          </Text>
+          <UploadButton
+            acceptedFileTypes={acceptedFileTypes}
+            isLoading={isLoading}
+            multiple={multiple}
+            onFileChange={onFileChange}
+            className={ComponentClassNames.FileUploaderDropZoneButton}
+            hiddenInput={hiddenInput}
+            onClick={onUploadButtonClick}
+          />
+        </UploadDropZone>
+        <Text className={ComponentClassNames.FileUploaderPreviewerText}>
+          {isSuccess ? (
+            <>
+              {uploadedFilesLength()} {translate('files uploaded')}
+            </>
+          ) : (
+            <>
+              {remainingFilesLength} {translate('files selected')}
+            </>
```

Comment:
I can use string literals here. I can't combine into 1 translate function, because but I can leave them as 2. 

```
{isSuccessful ? (
            <>{`${getUploadedFilesLength()} ${translate('files uploaded')}`}</>
          ) : (
            <>{`${remainingFilesLength} ${translate('files selected')}`}</>
          )}
```

Reply:
[89dfd48](https://github.com/aws-amplify/amplify-ui/pull/3028/commits/89dfd488afb30962645c5fdfe6a2d008a4b23934)